doc/user/content/overview/architecture.md

```diff
@@ -8,23 +8,97 @@ menu:
     weight: 10
 ---
 
-Materialize is a horizontally scalable, highly available distributed system. It separates storage and compute into different layers, allowing data ingestion, computation, and query serving to be **performed and scaled independently**.
+Materialize is a streaming database designed to provide correct, low latency answers to queries that need to be answered repeatedly by **incrementally maintaining answers** to those queries. Materialize enables users to ingest as much data as they want (by separating compute and storage into decoupled layers), perform as many queries as they want (by being horizontally and vertically scalable), while being highly available and flexible to operate (by offering active-active replication).
+
+
+The Materialize architecture is broken into three separate layers:
+* Storage: which is responsible for reading from and writing to external sources and sinks.
+* Compute: which is responsible for incrementally maintaining the results of user requested queries.
+* SQL Control Plane (Adapter): which is responsible for actually responding to user requests, and serving results back to users.
```

Comment:
do we still want to call it the Adapter in public facing docs or is it always meant to be called sql control plane?

Reply:
We don't really use the term "SQL Control Plane" anywhere other than in this one Marketing diagram, so...I'd rather stick with Adapter for now.

Reply:
done!